### PR TITLE
Add missing create_percent method

### PR DESCRIPTION
**Pull request by serwy**
See: https://github.com/enthought/traitsui/issues/487

### CODE REVIEW

File: pyface/ui/qt4/progress_dialog.py [hunk 1]
```diff
@@ -222,6 +222,12 @@ def _create_message(self, dialog, layout):
         self._message_control = label
         return
 
+    def _create_percent(self, dialog, parent_sizer):
+        if not self.show_percent:
+            return
+
+        raise NotImplementedError
```
Comment (corranwebster) [inline pyface/ui/qt4/progress_dialog.py]: I think I'd be tempted to log or warn and do nothing rather than raise here.

Comment (serwy) [reply pyface/ui/qt4/progress_dialog.py]: I copied the method as it exists in the `wx` backed. Are we going to fix it in both places?

Comment (corranwebster) [reply pyface/ui/qt4/progress_dialog.py]: If this is the same as Wx, it's fine.

**Comment by serwy** [comment]
@corranwebster Thank you for the review

**Pull request merged**